<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsGlobePluginDialogGuiBase</class>
 <widget class="QDialog" name="QgsGlobePluginDialogGuiBase">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>374</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Globe Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMap">
      <attribute name="title">
       <string>Map</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="mBaseLayerGroupBox">
         <property name="title">
          <string>Base &amp;Layer</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelBaseLayerSource">
            <property name="text">
             <string>Source</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelBaseLayerURL">
            <property name="text">
             <string>URL</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxBaseLayer"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditBaseLayerURL"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBoxSky">
         <property name="title">
          <string>Sk&amp;y</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="labelDateTime">
            <property name="text">
             <string>Date / Time (UTC)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateTimeEdit" name="dateTimeEditSky">
            <property name="displayFormat">
             <string>dd.MM.yyyy HH:mm</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxSkyAutoAmbient">
            <property name="text">
             <string>Auto ambient</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="horizontalSliderMinAmbient">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacerMapTab">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabElevation">
      <attribute name="title">
       <string>Elevation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButtonElevationLayerBrowse">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>27</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxElevationLayerType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Raster</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TMS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelElevationLayerType">
         <property name="text">
          <string>T&amp;ype</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxElevationLayerType</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelVerticalScale">
         <property name="text">
          <string>Vertical Scale</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditElevationLayerPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelElevationLayerPath">
         <property name="text">
          <string>URL/File</string>
         </property>
         <property name="buddy">
          <cstring>lineEditElevationLayerPath</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButtonElevationLayerUp">
           <property name="text">
            <string>Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonElevationLayerDown">
           <property name="text">
            <string>Down</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonElevationLayerAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonElevationLayerRemove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QTableWidget" name="elevationDatasourcesWidget">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::NoDragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::IgnoreAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Cache</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Path</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="mSpinBoxVerticalScale">
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNavigation">
      <attribute name="title">
       <string>Navigation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBoxScrolling">
         <property name="title">
          <string>Scrolling</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="2" column="1">
           <widget class="QSlider" name="sliderScrollSensitivity">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelScollSensitivity">
            <property name="text">
             <string>Sensitivity:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxInvertScroll">
            <property name="text">
             <string>Invert scroll wheel</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <spacer name="verticalSpacerNavigationTab">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabVideo">
      <attribute name="title">
       <string>Video</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="2" column="0">
        <widget class="QLabel" name="labelVideoInfo">
         <property name="text">
          <string>&lt;i&gt;Changing video settings requires a restart of the globe plugin&lt;/i&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacerVideoTab">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxAntiAliasing">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Anti Aliasing</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelAASamples">
            <property name="text">
             <string>Samples</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditAASamples">
            <property name="placeholderText">
             <string>[Leave empty for maximum]</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabStereo">
      <attribute name="title">
       <string>Stereo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelStereoMode">
         <property name="text">
          <string>Stereo Mode</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelScreenDistance">
         <property name="text">
          <string>Screen distance (m)</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxStereoScreenDistance</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QDoubleSpinBox" name="spinBoxStereoScreenDistance">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelScreenWidth">
         <property name="text">
          <string>Screen width (m)</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QDoubleSpinBox" name="spinBoxStereoScreenWidth">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QLabel" name="labelSplitStereoHorizontalSeparation">
         <property name="text">
          <string>Split stereo horizontal separation (px)</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QLabel" name="labelSplitStereoVerticalSeparation">
         <property name="text">
          <string>Split stereo vertical separation (px)</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0" colspan="2">
        <widget class="QLabel" name="labelBoxSplitStereoVerticalEyeMapping">
         <property name="text">
          <string>Split stereo vertical eye mapping</string>
         </property>
        </widget>
       </item>
       <item row="15" column="2">
        <widget class="QComboBox" name="comboBoxSplitStereoVerticalEyeMapping">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string notr="true">LEFT_EYE_TOP_VIEWPORT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">LEFT_EYE_BOTTOM_VIEWPORT</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QDoubleSpinBox" name="spinBoxStereoScreenHeight">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelScreenHeight">
         <property name="text">
          <string>Screen height (m)</string>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QSpinBox" name="spinBoxSplitStereoVerticalSeparation">
         <property name="accelerated">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QSpinBox" name="spinBoxSplitStereoHorizontalSeparation">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="comboBoxStereoMode"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelEyeSeparation">
         <property name="text">
          <string>Eye separation (m)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="spinBoxStereoEyeSeparation">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
        </widget>
       </item>
       <item row="16" column="2">
        <widget class="QPushButton" name="pushButtonStereoResetDefaults">
         <property name="text">
          <string>Reset to defaults</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="labelSplitStereoHorizontalEyeMapping">
         <property name="text">
          <string>Split stereo horizontal eye mapping</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QComboBox" name="comboBoxSplitStereoHorizontalEyeMapping">
         <item>
          <property name="text">
           <string notr="true">LEFT_EYE_LEFT_VIEWPORT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">LEFT_EYE_RIGHT_VIEWPORT</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBoxFeatureIdentification">
         <property name="text">
          <string>Enable feature identification</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxFrustumHighlighting">
         <property name="text">
          <string>Enable frustum highlighting</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacerAdvancedTab">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBoxStereoMode</tabstop>
  <tabstop>spinBoxStereoEyeSeparation</tabstop>
  <tabstop>spinBoxStereoScreenDistance</tabstop>
  <tabstop>spinBoxStereoScreenWidth</tabstop>
  <tabstop>spinBoxStereoScreenHeight</tabstop>
  <tabstop>spinBoxSplitStereoHorizontalSeparation</tabstop>
  <tabstop>comboBoxSplitStereoHorizontalEyeMapping</tabstop>
  <tabstop>spinBoxSplitStereoVerticalSeparation</tabstop>
  <tabstop>comboBoxSplitStereoVerticalEyeMapping</tabstop>
  <tabstop>pushButtonStereoResetDefaults</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
